Cubical/Homotopy/HSpace.agda

```diff
@@ -27,6 +34,89 @@ record AssocHSpace {ℓ : Level} {A : Pointed ℓ} (e : HSpace A) : Type ℓ whe
                (μ-assoc (pt A) y z)
                refl
 
+record LeftInvHSpace {ℓ : Level} {A : Pointed ℓ} (e : HSpace A) : Type ℓ where
```

Comment:
We put universe levels at the beginning of files (wasn't done in this file yet, so if you don't have the time to fix it, you could just leave it like that) as a private variable (see CONTRIBUTING.md). You can also do that to the second universe level ```you use below and ```A```.